starknet_py/net/account/account_client.py

```diff
@@ -743,3 +763,14 @@ def make_invoke_function_by_version(
 
 def get_account_version():
     return 1 if "__validate__" in COMPILED_ACCOUNT_CONTRACT else 0
+
+
+def sign_transactions(
```

Comment:
Probably a better place for this would be `BaseSigner.sign_transaction` on `many=True` argument passed